src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -48,4 +48,36 @@ trait DataAccess
       workflowFailureQuery.schema ++
       workflowMessageQuery.schema ++
       workflowAuditStatusQuery.schema
+
+  def truncateAll: WriteAction[Int] = {
+    // important to keep the right order for referential integrity !
```

Comment:
Alternative 1: determine referential integrity dynamically by creating a graph

Alternative 2: temporarily disable FOREIGN_KEY_CHECKS and delete in any order

But before doing these, we should decide whether this truncation approach is the right way to go.